components/collections/editRequest.vue

```diff
@@ -93,7 +93,7 @@ export default {
     };
   },
   watch: {
-    "requestUpdateData.collectionIndex": function resetFolderIndex() {
+    "requestUpdateData.collectionIndex":function resetFolderIndex() { //resetFolderIndex Function
```

Comment:
```suggestion
    "requestUpdateData.collectionIndex": function resetFolderIndex() {
```

Reply:
FYI this is a commit suggestion, just click on `commit suggestion` and you can commit the suggested change with ease.